atoms/badge/index.html

```diff
@@ -0,0 +1,181 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/index.css" />
+    <link rel="stylesheet" href="index.css" />
+    <link rel="stylesheet" href="/utils/utils.css" />
+    <link rel="stylesheet" href="/utils/prism.css" />
+    <title>thisUI</title>
+  </head>
+  <body data-prismjs-copy-timeout="500">
+    <header class="tuid__head">
+      <nav class="tuid__nav--top"></nav>
+    </header>
+
+    <div
+      class="tuid__drawer tui__dis--none invis tui__scrollbar--invisible"
+    ></div>
+    <main class="tuid__main tui__scrollbar--invisible">
+      <h1 class="tui__h-1 tui__m-tb-lg">Badge</h1>
+      Badges can be used to diaplay small amount of information which can draw
+      quick attention.
+      <ul>
+        <li>
+          <h3 class="tui__h-3 tui__m-tb-lg">Indicator Badge</h3>
+          <p class="tui__text-sm tui__m-b-sm">
+            Indicator badge is avialable in following different color variants:
+          </p>
+          <ul class="tui__bw-sm tui__pd-md tuid__li--component">
+            <li>
+              <div class="tuid__badge--display tui__pos--rel">
+                <div
+                  class="tui__badge-xs tui__bgcol--red tui__badge-pos-abs-tr"
+                ></div>
+              </div>
+            </li>
+            <li>
+              <div class="tuid__badge--display tui__pos--rel">
+                <div
+                  class="tui__badge-xs tui__bgcol--yellow tui__badge-pos-abs-tr"
+                >
+                  9
+                </div>
+              </div>
+            </li>
+            <li>
+              <div class="tuid__badge--display tui__pos--rel">
+                <div
+                  class="tui__badge-xs tui__bgcol--green tui__badge-pos-abs-tr"
+                >
+                  44
+                </div>
+              </div>
+            </li>
+            <li>
+              <div class="tuid__badge--display tui__pos--rel">
+                <div
+                  class="tui__badge-xs tui__bgcol--blue tui__badge-pos-abs-tr"
+                >
+                  444
+                </div>
+              </div>
+            </li>
+          </ul>
+          <div class="preformat tui__m-tb-lg">
+            <pre id><code class="language-markup">
+    <div class="tui__pos--rel">
+        <div
+        class="tui__badge-xs tui__bgcol--red tui__badge-pos-abs-tr">
+        </div>
+    </div>
+
+    <div class="tui__pos--rel">
+        <div
+        class="tui__badge-xs tui__bgcol--yellow tui__badge-pos-abs-tr">
+        9
+        </div>
+    </div>
+
+    <div class="tui__pos--rel">
+        <div
+        class="tui__badge-xs tui__bgcol--green tui__badge-pos-abs-tr">
+        44
+        </div>
+    </div>
+
+    <div class="tuid__badge--display tui__pos--rel">
```

Comment:
I just depicted while reviewing a minor fix is needed here. As tuid__badge--display is not a class the users will be able to access so we should remove it from here.
Moreover, this class is not related to any functionality it is just there to give background color and padding to the representation div which is used as a placeholder for the application of badges.

Reply:
I just made the changes that were needed